admin/commands/storage/read_range_cluster_blocks.go

```diff
@@ -0,0 +1,60 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/dgraph-io/badger/v2"
+
+	"github.com/onflow/flow-go/admin"
+	"github.com/onflow/flow-go/admin/commands"
+	"github.com/onflow/flow-go/cmd/util/cmd/read-block-light"
+	"github.com/onflow/flow-go/model/flow"
+	storage "github.com/onflow/flow-go/storage/badger"
+)
+
+var _ commands.AdminCommand = (*ReadRangeClusterBlocksCommand)(nil)
+
+type ReadRangeClusterBlocksCommand struct {
+	db       *badger.DB
+	headers  *storage.Headers
+	payloads *storage.ClusterPayloads
+}
+
+func NewReadRangeClusterBlocksCommand(db *badger.DB, headers *storage.Headers, payloads *storage.ClusterPayloads) commands.AdminCommand {
+	return &ReadRangeClusterBlocksCommand{
+		db:       db,
+		headers:  headers,
+		payloads: payloads,
+	}
+}
+
+func (c *ReadRangeClusterBlocksCommand) Handler(ctx context.Context, req *admin.CommandRequest) (interface{}, error) {
+	input, ok := req.Data.(map[string]interface{})
```

Comment:
parse chain id as part of parseHeightRangeRequestData?  you can pass in a requireChainID bool to the function to indicate if this should error or not

Reply:
`parseHeightRangeRequestData` is being reused by other admin command. So I extracted a parseString to be reused